kytos/core/napps/manager.py

```diff
@@ -90,7 +89,6 @@ def install(self, napp_uri, enable=True):
             dst = self._installed / napp.username / napp.name
             self._create_module(dst.parent)
             shutil.move(str(napp_folder), str(dst))
-            os.chmod(dst, mode=0o755)
```

Comment:
If I understood, we are:

1. Creating a tmp dir without fixing a proper permission (each user has your own umask);
2. Moving this folder to destination;

So yes, we need to force a chmod here, because we cannot make sure what is the tmp permissions, right ?